guides/common/modules/proc_updating-disconnected-server.adoc

```diff
@@ -0,0 +1,163 @@
+* {RepoRHEL8AppStream}
+* {RepoRHEL8ServerSatelliteServerProductVersion}
+* {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+
+. Download the debug certificate of the organization and store it locally at `/etc/pki/katello/certs/org-debug-cert.pem` or a location of your choosing.
+For more information, see {AdministeringDocURL}Creating_an_Organization_Debug_Certificate_admin[Creating an Organization Debug Certificate] in _{AdministeringDocTitle}_.
+
+. Create a Yum configuration file under `/etc/yum.repos.d` with the following repository information:
++
+[options="nowrap" subs="attributes"]
+----
+[{RepoRHEL8BaseOS}]
+name=Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel8/8/x86_64/baseos/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+
+[{RepoRHEL8AppStream}]
+name=Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel8/8/x86_64/appstream/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+
+[{RepoRHEL8ServerSatelliteServerProductVersion}]
+name={ProjectName} {ProjectVersion} for RHEL 8 RPMs x86_64
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/layered/rhel8/x86_64/satellite/{ProjectVersion}/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+
+[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
+name={ProjectName} Maintenance {ProjectVersion} for RHEL 8 RPMs x86_64
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/layered/rhel8/x86_64/sat-maintenance/{ProjectVersion}/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+----
++
+. In the configuration file, complete the following steps:
+* For the `sslclientcert` and `sslclientkey` options, replace `/etc/pki/katello/certs/org-debug-cert.pem` with the location of the downloaded organization debug certificate.
+* For the `{foreman-example-com}` option, specify the correct FQDN for your deployment.
+* For the `baseurl` option, replace `My_Organization` with the correct organization label.
+
+. Obtain the organization label:
++
+----
+# hammer organization list
+----
+
+. Enter the `reposync` command:
++
+[options="nowrap" subs="attributes"]
+----
+# reposync --delete --download-metadata -p ~/{project-context}-repos -n \
```

Comment:
reposync is not installed by default, so that needs a specific instruction as well. I'd actually advise to use `dnf reposync` which can be installed by `dnf install 'dnf-command(reposync)'`. If you insist on using `reposync` then you need to install `yum-utils`.

Reply:
Considered `reposync` as required. Addressed :+1: